project/default.nix

```diff
@@ -97,7 +104,7 @@ in
   # Set to false to disable building the hoogle database when entering
   # the nix-shell.
 
-, android ? throw "No Android config"
+, android ? {}
```

Comment:
@ryantrinkle I think I'm ok with getting rid of these more descriptive error messages. Now if a user tries to build `android.frontend` when they didn't specify an android config, they'll get `error: attribute ‘frontend’ in selection path ‘android.frontend’ not found`

Reply:
Sounds good to me